Khangtph verify the result 0 matches is displayed in car finder find your perfect car kelley blue book page

truongkhang0905 opened the pull request:
Khangtph verify the result 0 matches is displayed in car finder find your perfect car kelley blue book page

- Delete target folder
- Create CarFinderPage folder include in CarFinderPage file and ShopByPricePage file
- Create CarFinderPageTests folder include in CarFinderPageTests file

Commit:
- Delete target folder and some files are not relative to project
- Create CarFinderPages folder include in CarFinderPage and ShopByPricePage file
- Create CarFinderPagetests include in CarFinderPageTest file

truongkhang0905 committed:
Delete demo.java

Commit:
Delete all this file

Commit:
Merge branch 'KhangTPH-verify-the-result-0-matches-is-displayed-in-Car-Finder-Find-Your-Perfect-Car-Kelley-Blue-Book-page' of https://github.com/namnguyen-itvn/hcm-aut04 into KhangTPH-verify-the-result-0-matches-is-displayed-in-Car-Finder-Find-Your-Perfect-Car-Kelley-Blue-Book-page

namnguyen-itvn commented:
@MrJade267 pls review this PR

MrJade267 reviewed (commented):
Coding clear
The way to set the function is not good, only lowercase the first letter
Test run OK

Commit:
Delete target file
Edit:
 - Customkeyword file
 - Datadriven file
 - Set up web element outside function/method

namnguyen-itvn committed:
Merge branch 'develop' into KhangTPH-verify-the-result-0-matches-is-displayed-in-Car-Finder-Find-Your-Perfect-Car-Kelley-Blue-Book-page

namnguyen-itvn committed:
mergedevelop and resolve conflict

namnguyen-itvn committed:
Merge remote-tracking branch 'origin/KhangTPH-verify-the-result-0-matches-is-displayed-in-Car-Finder-Find-Your-Perfect-Car-Kelley-Blue-Book-page' into KhangTPH-verify-the-result-0-matches-is-displayed-in-Car-Finder-Find-Your-Perfect-Car-Kelley-Blue-Book-page
